.github/workflows/stale.yml

```diff
@@ -10,8 +10,10 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        debug-only: false
-        days-before-stale: -1
+        debug-only: true
         exempt-issue-labels: "blue-ticket, p1-urgent, p2-high, p3-medium, p4-low"
+        days-before-stale: 30
+        stale-issue-message: "This issue is stale because it has been open 30 days with no activity. Remove the stale label or comment or this will be closed in 7 days."
```

Comment:
```suggestion
        stale-issue-message: "This issue was marked stale because it has been open for 30 days with no activity. Remove the stale label or comment or this will be closed in 7 days."
```